libraries/AP_VisualOdom/AP_VisualOdom_Backend.cpp

```diff
@@ -56,7 +56,7 @@ void AP_VisualOdom_Backend::handle_vision_position_delta_msg(const mavlink_messa
     _last_update_ms = now_ms;
 
     // send to EKF
-    const float time_delta_sec = packet.time_delta_usec / 1000000.0f;
+    const float time_delta_sec = packet.time_delta_usec * 0.000001f;
```

Comment:
This one is wrong as we are outside float limit

Reply:
so would it be better to do two multiplies? *.001 * .001  is that still faster than a float divide?

Reply:
Rather than just blindly converting all divides to multiply, is someone looking at where these are done? Fixing one calculation in a tight loop is worth thousands on individual fixes that are rarely executed.

Reply:
it would be better as 1.0e-6

Reply:
Yes, [Amdahl's law](https://en.wikipedia.org/wiki/Amdahl%27s_law) is very important, but people are habit animals, and programing patterns do get propagated by copy-paste :) So a good pattern is better

Reply:
Fixed